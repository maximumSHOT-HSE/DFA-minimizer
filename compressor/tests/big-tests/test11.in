106 1166 11
1 67 0
1 26 1
1 85 2
1 33 3
1 3 4
1 23 5
1 55 6
1 65 7
1 42 8
1 90 9
1 79 10
2 64 0
2 72 1
2 92 2
2 40 3
2 78 4
2 84 5
2 11 6
2 37 7
2 24 8
2 77 9
2 84 10
3 39 0
3 29 1
3 35 2
3 36 3
3 34 4
3 96 5
3 7 6
3 2 7
3 89 8
3 14 9
3 35 10
4 56 0
4 53 1
4 19 2
4 82 3
4 57 4
4 50 5
4 22 6
4 21 7
4 75 8
4 9 9
4 10 10
5 50 0
5 54 1
5 48 2
5 17 3
5 45 4
5 82 5
5 28 6
5 37 7
5 72 8
5 32 9
5 9 10
6 8 0
6 3 1
6 87 2
6 50 3
6 53 4
6 12 5
6 36 6
6 26 7
6 67 8
6 45 9
6 20 10
7 9 0
7 40 1
7 9 2
7 48 3
7 24 4
7 73 5
7 36 6
7 6 7
7 1 8
7 48 9
7 15 10
8 88 0
8 63 1
8 88 2
8 95 3
8 39 4
8 38 5
8 90 6
8 91 7
8 2 8
8 46 9
8 12 10
9 67 0
9 63 1
9 94 2
9 42 3
9 26 4
9 23 5
9 72 6
9 87 7
9 90 8
9 95 9
9 96 10
10 65 0
10 4 1
10 37 2
10 86 3
10 10 4
10 72 5
10 31 6
10 82 7
10 7 8
10 75 9
10 93 10
11 20 0
11 58 1
11 12 2
11 56 3
11 73 4
11 84 5
11 47 6
11 95 7
11 54 8
11 12 9
11 37 10
12 18 0
12 42 1
12 19 2
12 48 3
12 85 4
12 8 5
12 49 6
12 28 7
12 88 8
12 18 9
12 48 10
13 11 0
13 1 1
13 67 2
13 21 3
13 20 4
13 32 5
13 7 6
13 17 7
13 33 8
13 41 9
13 66 10
14 77 0
14 24 1
14 61 2
14 60 3
14 14 4
14 83 5
14 18 6
14 85 7
14 63 8
14 59 9
14 33 10
15 40 0
15 59 1
15 58 2
15 14 3
15 23 4
15 48 5
15 42 6
15 2 7
15 84 8
15 82 9
15 54 10
16 94 0
16 41 1
16 4 2
16 4 3
16 77 4
16 15 5
16 30 6
16 14 7
16 2 8
16 10 9
16 87 10
17 40 0
17 64 1
17 84 2
17 50 3
17 49 4
17 87 5
17 27 6
17 34 7
17 84 8
17 44 9
17 35 10
18 18 0
18 3 1
18 13 2
18 79 3
18 18 4
18 67 5
18 48 6
18 6 7
18 62 8
18 75 9
18 12 10
19 3 0
19 28 1
19 20 2
19 24 3
19 84 4
19 4 5
19 75 6
19 11 7
19 26 8
19 22 9
19 95 10
20 23 0
20 91 1
20 39 2
20 62 3
20 94 4
20 74 5
20 4 6
20 26 7
20 46 8
20 74 9
20 3 10
21 31 0
21 23 1
21 5 2
21 11 3
21 86 4
21 10 5
21 68 6
21 34 7
21 31 8
21 31 9
21 80 10
22 43 0
22 32 1
22 41 2
22 9 3
22 18 4
22 52 5
22 97 6
22 85 7
22 51 8
22 49 9
22 6 10
23 55 0
23 70 1
23 1 2
23 77 3
23 50 4
23 59 5
23 2 6
23 49 7
23 92 8
23 33 9
23 50 10
24 3 0
24 62 1
24 4 2
24 89 3
24 33 4
24 85 5
24 95 6
24 17 7
24 81 8
24 29 9
24 23 10
25 42 0
25 69 1
25 17 2
25 85 3
25 88 4
25 9 5
25 59 6
25 68 7
25 6 8
25 65 9
25 18 10
26 93 0
26 55 1
26 1 2
26 31 3
26 61 4
26 76 5
26 25 6
26 60 7
26 46 8
26 92 9
26 49 10
27 9 0
27 86 1
27 69 2
27 46 3
27 82 4
27 42 5
27 89 6
27 38 7
27 50 8
27 27 9
27 77 10
28 58 0
28 59 1
28 97 2
28 52 3
28 89 4
28 15 5
28 19 6
28 17 7
28 63 8
28 43 9
28 9 10
29 45 0
29 90 1
29 87 2
29 67 3
29 51 4
29 39 5
29 41 6
29 22 7
29 54 8
29 64 9
29 54 10
30 34 0
30 37 1
30 15 2
30 9 3
30 74 4
30 27 5
30 42 6
30 77 7
30 95 8
30 23 9
30 44 10
31 95 0
31 51 1
31 49 2
31 47 3
31 67 4
31 46 5
31 95 6
31 17 7
31 46 8
31 59 9
31 47 10
32 88 0
32 82 1
32 57 2
32 78 3
32 54 4
32 25 5
32 2 6
32 21 7
32 50 8
32 21 9
32 5 10
33 40 0
33 24 1
33 24 2
33 69 3
33 34 4
33 47 5
33 33 6
33 41 7
33 15 8
33 85 9
33 7 10
34 13 0
34 48 1
34 41 2
34 84 3
34 20 4
34 1 5
34 55 6
34 35 7
34 47 8
34 48 9
34 9 10
35 11 0
35 74 1
35 20 2
35 2 3
35 80 4
35 83 5
35 34 6
35 29 7
35 23 8
35 21 9
35 77 10
36 26 0
36 22 1
36 81 2
36 83 3
36 63 4
36 77 5
36 33 6
36 44 7
36 22 8
36 1 9
36 67 10
37 9 0
37 12 1
37 68 2
37 32 3
37 70 4
37 59 5
37 16 6
37 20 7
37 22 8
37 72 9
37 15 10
38 31 0
38 21 1
38 40 2
38 17 3
38 23 4
38 2 5
38 18 6
38 28 7
38 79 8
38 27 9
38 72 10
39 87 0
39 48 1
39 64 2
39 64 3
39 29 4
39 21 5
39 68 6
39 76 7
39 83 8
39 47 9
39 71 10
40 5 0
40 3 1
40 34 2
40 22 3
40 11 4
40 64 5
40 42 6
40 60 7
40 26 8
40 43 9
40 39 10
41 90 0
41 37 1
41 9 2
41 22 3
41 15 4
41 9 5
41 62 6
41 52 7
41 1 8
41 45 9
41 62 10
42 53 0
42 71 1
42 69 2
42 20 3
42 83 4
42 19 5
42 59 6
42 65 7
42 85 8
42 80 9
42 41 10
43 46 0
43 65 1
43 51 2
43 1 3
43 17 4
43 83 5
43 64 6
43 66 7
43 3 8
43 1 9
43 29 10
44 58 0
44 52 1
44 53 2
44 32 3
44 75 4
44 47 5
44 97 6
44 46 7
44 78 8
44 69 9
44 6 10
45 13 0
45 92 1
45 11 2
45 57 3
45 60 4
45 85 5
45 74 6
45 73 7
45 26 8
45 40 9
45 93 10
46 97 0
46 91 1
46 63 2
46 54 3
46 89 4
46 53 5
46 85 6
46 5 7
46 34 8
46 89 9
46 91 10
47 68 0
47 81 1
47 76 2
47 79 3
47 4 4
47 28 5
47 76 6
47 44 7
47 55 8
47 94 9
47 82 10
48 64 0
48 15 1
48 12 2
48 24 3
48 64 4
48 57 5
48 36 6
48 67 7
48 60 8
48 75 9
48 70 10
49 18 0
49 83 1
49 77 2
49 41 3
49 46 4
49 76 5
49 30 6
49 33 7
49 88 8
49 29 9
49 49 10
50 34 0
50 80 1
50 51 2
50 90 3
50 39 4
50 3 5
50 3 6
50 32 7
50 10 8
50 37 9
50 46 10
51 41 0
51 66 1
51 88 2
51 54 3
51 41 4
51 5 5
51 63 6
51 26 7
51 5 8
51 7 9
51 19 10
52 38 0
52 13 1
52 97 2
52 19 3
52 63 4
52 23 5
52 3 6
52 61 7
52 50 8
52 57 9
52 85 10
53 78 0
53 74 1
53 30 2
53 62 3
53 94 4
53 70 5
53 21 6
53 7 7
53 10 8
53 89 9
53 91 10
54 18 0
54 96 1
54 27 2
54 78 3
54 60 4
54 14 5
54 39 6
54 70 7
54 42 8
54 24 9
54 5 10
55 83 0
55 33 1
55 25 2
55 80 3
55 77 4
55 90 5
55 22 6
55 58 7
55 46 8
55 34 9
55 71 10
56 9 0
56 51 1
56 10 2
56 3 3
56 21 4
56 10 5
56 90 6
56 19 7
56 28 8
56 45 9
56 12 10
57 68 0
57 7 1
57 73 2
57 34 3
57 49 4
57 90 5
57 92 6
57 56 7
57 67 8
57 84 9
57 64 10
58 69 0
58 12 1
58 8 2
58 77 3
58 70 4
58 7 5
58 69 6
58 4 7
58 2 8
58 97 9
58 24 10
59 72 0
59 96 1
59 29 2
59 32 3
59 23 4
59 38 5
59 52 6
59 38 7
59 15 8
59 16 9
59 72 10
60 72 0
60 23 1
60 5 2
60 15 3
60 12 4
60 93 5
60 72 6
60 22 7
60 25 8
60 51 9
60 34 10
61 60 0
61 72 1
61 5 2
61 27 3
61 17 4
61 18 5
61 9 6
61 53 7
61 81 8
61 61 9
61 59 10
62 63 0
62 47 1
62 42 2
62 4 3
62 7 4
62 69 5
62 42 6
62 53 7
62 4 8
62 54 9
62 28 10
63 92 0
63 32 1
63 97 2
63 47 3
63 27 4
63 4 5
63 6 6
63 5 7
63 23 8
63 44 9
63 87 10
64 7 0
64 86 1
64 59 2
64 96 3
64 72 4
64 65 5
64 84 6
64 85 7
64 72 8
64 73 9
64 14 10
65 52 0
65 94 1
65 60 2
65 40 3
65 9 4
65 87 5
65 71 6
65 67 7
65 89 8
65 62 9
65 68 10
66 22 0
66 68 1
66 51 2
66 97 3
66 73 4
66 84 5
66 36 6
66 42 7
66 16 8
66 56 9
66 1 10
67 50 0
67 90 1
67 33 2
67 84 3
67 83 4
67 3 5
67 38 6
67 63 7
67 20 8
67 39 9
67 96 10
68 30 0
68 2 1
68 86 2
68 16 3
68 11 4
68 59 5
68 38 6
68 17 7
68 67 8
68 50 9
68 62 10
69 23 0
69 80 1
69 8 2
69 42 3
69 85 4
69 83 5
69 49 6
69 20 7
69 32 8
69 13 9
69 4 10
70 43 0
70 97 1
70 3 2
70 90 3
70 9 4
70 75 5
70 82 6
70 32 7
70 38 8
70 29 9
70 93 10
71 51 0
71 90 1
71 73 2
71 8 3
71 24 4
71 2 5
71 32 6
71 18 7
71 32 8
71 57 9
71 70 10
72 84 0
72 35 1
72 42 2
72 7 3
72 9 4
72 17 5
72 13 6
72 74 7
72 44 8
72 9 9
72 46 10
73 58 0
73 34 1
73 90 2
73 88 3
73 39 4
73 95 5
73 68 6
73 95 7
73 7 8
73 15 9
73 43 10
74 49 0
74 23 1
74 39 2
74 51 3
74 56 4
74 82 5
74 83 6
74 8 7
74 22 8
74 54 9
74 66 10
75 61 0
75 9 1
75 13 2
75 59 3
75 80 4
75 51 5
75 62 6
75 37 7
75 77 8
75 23 9
75 7 10
76 13 0
76 91 1
76 90 2
76 35 3
76 79 4
76 5 5
76 74 6
76 29 7
76 16 8
76 23 9
76 9 10
77 68 0
77 63 1
77 95 2
77 66 3
77 50 4
77 46 5
77 56 6
77 67 7
77 87 8
77 62 9
77 82 10
78 5 0
78 7 1
78 41 2
78 20 3
78 2 4
78 70 5
78 50 6
78 33 7
78 19 8
78 16 9
78 18 10
79 34 0
79 66 1
79 70 2
79 48 3
79 64 4
79 74 5
79 71 6
79 71 7
79 85 8
79 12 9
79 89 10
80 68 0
80 76 1
80 8 2
80 31 3
80 54 4
80 39 5
80 16 6
80 54 7
80 69 8
80 5 9
80 41 10
81 30 0
81 33 1
81 21 2
81 1 3
81 97 4
81 21 5
81 85 6
81 5 7
81 22 8
81 20 9
81 47 10
82 45 0
82 26 1
82 78 2
82 59 3
82 41 4
82 42 5
82 82 6
82 63 7
82 63 8
82 89 9
82 10 10
83 1 0
83 44 1
83 88 2
83 49 3
83 45 4
83 82 5
83 88 6
83 96 7
83 47 8
83 9 9
83 29 10
84 67 0
84 23 1
84 1 2
84 58 3
84 55 4
84 11 5
84 87 6
84 63 7
84 50 8
84 90 9
84 58 10
85 94 0
85 25 1
85 59 2
85 37 3
85 68 4
85 85 5
85 47 6
85 86 7
85 82 8
85 7 9
85 70 10
86 26 0
86 50 1
86 85 2
86 20 3
86 24 4
86 90 5
86 57 6
86 31 7
86 97 8
86 67 9
86 77 10
87 96 0
87 61 1
87 17 2
87 12 3
87 97 4
87 74 5
87 21 6
87 44 7
87 78 8
87 22 9
87 50 10
88 3 0
88 23 1
88 43 2
88 71 3
88 71 4
88 95 5
88 88 6
88 74 7
88 23 8
88 71 9
88 26 10
89 4 0
89 40 1
89 19 2
89 58 3
89 85 4
89 5 5
89 75 6
89 67 7
89 93 8
89 57 9
89 47 10
90 92 0
90 34 1
90 72 2
90 56 3
90 80 4
90 45 5
90 85 6
90 33 7
90 33 8
90 8 9
90 19 10
91 51 0
91 79 1
91 61 2
91 32 3
91 97 4
91 43 5
91 24 6
91 46 7
91 69 8
91 65 9
91 76 10
92 38 0
92 45 1
92 60 2
92 59 3
92 70 4
92 60 5
92 77 6
92 96 7
92 85 8
92 96 9
92 58 10
93 91 0
93 7 1
93 81 2
93 92 3
93 24 4
93 64 5
93 52 6
93 56 7
93 2 8
93 3 9
93 26 10
94 48 0
94 27 1
94 73 2
94 27 3
94 59 4
94 77 5
94 9 6
94 8 7
94 8 8
94 86 9
94 25 10
95 79 0
95 59 1
95 9 2
95 45 3
95 33 4
95 72 5
95 90 6
95 80 7
95 87 8
95 95 9
95 22 10
96 89 0
96 64 1
96 35 2
96 26 3
96 89 4
96 93 5
96 50 6
96 34 7
96 19 8
96 4 9
96 29 10
97 95 0
97 51 1
97 77 2
97 78 3
97 55 4
97 89 5
97 49 6
97 14 7
97 4 8
97 10 9
97 11 10
98 79 0
98 59 1
98 9 2
98 45 3
98 33 4
98 72 5
98 90 6
98 80 7
98 87 8
98 95 9
98 22 10
99 34 0
99 66 1
99 70 2
99 48 3
99 64 4
99 74 5
99 71 6
99 71 7
99 85 8
99 12 9
99 89 10
100 3 0
100 28 1
100 20 2
100 24 3
100 84 4
100 4 5
100 75 6
100 11 7
100 26 8
100 22 9
100 95 10
101 65 0
101 4 1
101 37 2
101 86 3
101 10 4
101 72 5
101 31 6
101 82 7
101 7 8
101 75 9
101 93 10
102 64 0
102 72 1
102 92 2
102 40 3
102 78 4
102 84 5
102 11 6
102 37 7
102 24 8
102 77 9
102 84 10
103 58 0
103 59 1
103 97 2
103 52 3
103 89 4
103 15 5
103 19 6
103 17 7
103 63 8
103 43 9
103 9 10
104 83 0
104 33 1
104 25 2
104 80 3
104 77 4
104 90 5
104 22 6
104 58 7
104 46 8
104 34 9
104 71 10
105 84 0
105 35 1
105 42 2
105 7 3
105 9 4
105 17 5
105 13 6
105 74 7
105 44 8
105 9 9
105 46 10
106 65 0
106 4 1
106 37 2
106 86 3
106 10 4
106 72 5
106 31 6
106 82 7
106 7 8
106 75 9
106 93 10
46
101
64 66 106 105 60 50 3 70 98 52 89 58 4 20 15 19 95 104 99 96 86 28 93 36 26 76 77 84 39 100 43 101 25 7 47 30 73 35 78 67 17 65 8 12 23 29 90 80 103 18 79 88 97 11 44 51 22 38 94 2 10 5 75 69 45 21 81 31 83 41 53 102 1 56 72 40 54 33 68 85 71 59 61 49 82 37 48 91 63 6 92 74 42 16 57 13 55 27 87 24 32 
