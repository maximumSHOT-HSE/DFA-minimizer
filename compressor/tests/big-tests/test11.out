15 90 6
3 1 0
6 1 0
6 1 1
2 1 3
10 1 4
14 1 5
10 2 1
9 2 2
1 2 4
2 2 4
3 2 5
12 2 5
2 3 0
11 3 1
13 3 1
2 3 2
3 3 2
6 3 4
9 3 4
6 3 5
9 4 1
14 4 1
6 4 2
8 4 3
14 4 3
12 4 4
15 4 4
8 4 5
1 5 0
8 5 0
15 5 0
2 5 1
4 5 2
13 5 2
1 5 3
5 5 3
4 5 4
7 5 5
4 6 0
7 6 1
11 6 2
15 6 2
3 6 4
7 6 4
11 6 5
11 7 0
12 7 1
1 7 2
12 7 3
1 7 5
4 7 5
15 7 5
13 8 0
7 8 2
5 8 5
9 8 5
10 9 0
15 9 1
13 9 4
7 10 0
3 10 1
8 10 1
3 10 3
7 10 3
11 10 3
5 10 4
14 11 0
1 11 1
5 11 1
5 11 2
8 11 2
4 11 3
8 11 4
9 12 0
12 12 2
11 12 4
14 12 4
14 13 2
10 13 3
12 14 0
6 14 3
2 14 5
10 14 5
5 15 0
4 15 1
10 15 2
9 15 3
13 15 3
15 15 3
13 15 5
6
5
5 6 8 11 13 
